app/assets/javascripts/selectors/done.js

```diff
@@ -0,0 +1,13 @@
+import moment from "moment";
+
+export const mountPastIterations = (pastIterations) => {
+  const sprints = pastIterations.map((sprint, index) => ({
+    ...sprint,
+    number: index + 1,
+    startDate: moment(sprint.startDate).format("ddd MMM Do Y"),
```

Comment:
depending on the locale assigned (pt for exemple), the formal output may be something like `Qui dez 6º 2018​​​​​`, in which looks a bit weird. What do you think?

Also, IMO it is not good to add/deal with `moment` or any text formatting in redux selectors. I feel that it is something more react component related and might limit past iterations formatting later. I believe that `startDate` and `endDate` should be in ISO here

Reply:
Thank for the review, Lucas! Yep, great idea, I gonna do this in another PR to change the dates in considering of user locale removing this logic to React component.

Reply:
Ref: [23314](https://central.cm42.io/projects/central-board-v2#story-23314)